cookbook/integrations/in_container.mk

```diff
@@ -11,10 +11,6 @@ $(SERIALIZED_PB_OUTPUT_DIR): clean
 serialize: $(SERIALIZED_PB_OUTPUT_DIR)
 	pyflyte --config /root/sandbox.config serialize workflows -f $(SERIALIZED_PB_OUTPUT_DIR)
 
-.PHONY: register
```

Comment:
I see you removed fast_register from some of the in_container files but not all of them... can we make sure they look exactly the same?

Reply:
i removed both command in `cookbook/in.container`

Reply:
done